buildSrc/src/main/kotlin/com/datadog/gradle/config/MavenConfig.kt

```diff
@@ -114,7 +114,7 @@ fun Project.publishingConfig(projectDescription: String) {
         signingExtension.apply {
             val privateKey = System.getenv("GPG_PRIVATE_KEY")
             val password = System.getenv("GPG_PASSWORD")
-            isRequired = true
+            isRequired = !hasProperty("dd-skip-signing")
```

Comment:
should be reverted

Reply:
I need this here for `publishToMavenLocal` to work. Is this what you were asking ?